improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -47,186 +47,118 @@
+                                 0.0, 0.0, 0.0]).reshape((1, 1, 3, 3))
+        cloud_low = (
+            set_up_probability_threshold_cube(
+                data_cld_low,
+                'low_type_cloud_area_fraction',
+                '1',
+                spp__relative_to_threshold='above',
+                forecast_thresholds=np.array([0.85])))
+
+        data_vis = np.array([0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0,
+                             0.0, 0.0, 1.0, 1.0, 0.0, 0.0,
+                             0.0, 1.0, 0.0]).reshape((2, 1, 3, 3))
+        visibility = (
+            set_up_probability_threshold_cube(
+                data_vis,
+                'visibility_in_air',
+                'm',
+                spp__relative_to_threshold='below',
+                forecast_thresholds=np.array([1000.0, 5000.0])))
+
+        data_snowv = np.array([0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
+                               0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
+                               0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
+                               0.0, 0.0, 0.0]).reshape((3, 1, 3, 3))
+        snowfall_vicinity = (
+            set_up_probability_threshold_cube(
+                data_snowv,
+                'lwe_snowfall_rate_in_vicinity',
+                'm s-1',
+                spp__relative_to_threshold='above',
+                forecast_thresholds=np.array([8.33333333e-09,
+                                              2.77777778e-08,
+                                              2.77777778e-07])))
+
+        data_rainv = np.array([0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 1.0, 1.0,
+                               0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 1.0,
+                               0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
+                               0.0, 0.0, 0.0]).reshape((3, 1, 3, 3))
+        rainfall_vicinity = (
+            set_up_probability_threshold_cube(
+                data_rainv,
+                'rainfall_rate_in_vicinity',
+                'm s-1',
+                spp__relative_to_threshold='above',
+                forecast_thresholds=np.array([8.33333333e-09,
+                                              2.77777778e-08,
+                                              2.77777778e-07])))
+
+        data_lightning = np.zeros((1, 1, 3, 3))
+        data_lightning[0, 0, 0, 0] = 0.25
+        data_lightning[0, 0, 0, 1] = 0.30
+        lightning = (
+            set_up_probability_threshold_cube(
+                data_lightning,
+                'number_of_lightning_flashes_per_unit_area_in_vicinity',
+                'm-2',
+                spp__relative_to_threshold='above',
+                forecast_thresholds=np.array([0.0])))
+
+        self.cubes = iris.cube.CubeList([
+            snowfall_rate, rainfall_rate, snowfall_vicinity, rainfall_vicinity,
+            cloud, cloud_low, visibility, lightning])
+        all = [cube.name() for cube in self.cubes]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'all'](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4957787)